conjur/api/ssl_client.py

```diff
@@ -20,7 +20,7 @@
 }
 
 # pylint: disable=too-few-public-methods
-class SSLService:
+class SSLClient:
     """
     SSLService
```

Comment:
GENERAL NOTE
Please change all docstrings to their appropriate, updated names. For example, SSLService here should now be SSLClient

Reply:
Changed docstring to SSLClient.
These is the only class its name is changed